src/overlays/actors/ovl_Bg_Jya_Megami/z_bg_jya_megami.h

```diff
@@ -4,11 +4,28 @@
 #include <ultra64.h>
 #include <global.h>
 
+#define BGJYAMEGAMI_NUMPIECES 13
+
 struct BgJyaMegami;
 
+typedef void (*BgJyaMegamiActionFunc)(struct BgJyaMegami*, GlobalContext*);
+
+typedef struct BgJyaMegami_Piece {
```

Comment:
`struct`s should only be named when necessary.
```suggestion
typedef struct {
```